SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,21 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.
+
+I'm sure you've heard about the threat caused by the COVID-19 coronavirus. and you've likely heard of many large events being cancelled due to concerns of spreading of the virus.
+
+At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
```

Comment:
I don't this paragraph is a good thing to say, the only thing the teachers will get out of it is that we may well cancel, and therefore they might pull out to save on hassle.

Reply:
I disagree. Most teams seem pretty comfortable with the fact that we'd only cancel if we had to, and by that point things would be pretty bad.

Reassurance that we're following PHE also can't hurt.

Reply:
Ok, perhaps I should have commented on sentences separately. I agree with the 2nd sentence of this, not the ones either side. Having regularly dealt with several team based events each year, whenever a comment around not cancelling is made normally I see teams fall away rather than feel assured.

I feel that this paragraph should instead read:
"We're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. Considering this is an event for minors, we will be taking decisions with an abundance of caution."